src/components/RequestDetails/SeeRequest.tsx

```diff
@@ -27,6 +28,14 @@ export const SeeRequest = ({ route: { params: p } }: RequestsNavigationProps<'SE
     Analytics().track('REQUEST_OPENED', { request: { ...p } })
   }, [p])
 
+  useBackHandler(() => {
+    if (prevScreen) {
+      navigation.navigate(prevScreen)
+      return true
+    }
+    return false
+  })
```

Comment:
This logic is repeated in several places. Please move it to custom hook e.g `usePrevScreenBackHandler`